go/enclave/rollupmanager/rollupmanager.go

```diff
@@ -129,9 +129,30 @@ func (re *rollupManager) CreateRollup() (*core.Rollup, error) {
 		return nil, fmt.Errorf("current head batch matches the rollup head bash")
 	}
 
-	newRollup := createNextRollup(rollup, batches)
+	limiter := RollupLimiter(MaxTransactionSize)
```

Comment:
can we have this rollupLimiter as a field in the rollupManager.
And then after each batch, you call something, and if the latest batch fills the rollup, then signal that to the host?

Reply:
I'm planning to do this after the cadence and refactor PRs go in, at this point signaling a rollup is full is a bit premature as the code is super tangled up; Batches can have different sizes according to which block they were applied due to cross chain messages (or lack of in case a dependency went missing) so the functionality for signaling a rollup is full needs to be aware of a bit more context